documentation/design_docs/questionnaire_guidelines

```diff
@@ -0,0 +1,41 @@
+- Is the question user friendly and does it use any language that might need an explanation?
+- Is the question impartial?
+- Is the 'question type' the most appropriate? 
+
+N.B. The 'question type' determines what type of answer the user can give in the Shiny app. Examples include 'text' (open-ended answer), 'numeric' (numeric answers only), 'select' (drop down box with prespecified answers), and 'slider' (an interactive scale to select a numeric value), etc.
+
+## Answer Style
+Asking a great question means nothing to us without a great answer, so we must carefully consider the answer options offered to the user to ensure we extract the most accurate information possible. When considering which answers should be offered to the user, one should consider:
+- Are the answers presented an appropriate response to the question asked?
+- Is the range of answers too limited or narrow? Are any options/values missing?
+- Is the range of answers too broad or ambiguous? Are there answers that overlap eachother?
+- Is the user able to answer in the most appropriate way? Would a different 'question type' be more suitable?
+
+## Storing Questions and Answers in the database
+Once the questions and corresponding answers have been decided, the next step is to store them in the questionnaire databse. The questionnaire database is called 'q_database' and can be found at app/modules/questionnaire/q_database. It's a csv file and follows a set structure, described below. 
+
+For each question please describe:
+- **q_number**: the question number, ie question number 1 means the first question asked to the user. If rearranging numbers, ensure to re-label every question so there are no duplicated or missing numbers.
+- **q_type**: the 'question type' as referred to above. This will determine the type of answer offered to the user.
+- **q_variable**: the question variable, ie the name of the questio in the UI. This should be decriptive but very concise and adhere to the coding guidelines, eg drug_type.
+- **q_text**: the question text is what will be shown to the user, eg 'What type of drug are you testing?'. Make sure this is a full sentence and is grammatically correct.
+- **answer_type**: the type of answer that is required by the question. This could be categorical (Yes, No), numeric (ie the prior variance), numeric_bounded (used mainly for sliders where there is a maximum value), or comma-separated list (for text boxes to record toxicity profiles).
+- **params**: answers should be recorded in this column in a very specific way. 
+  - For categorical answer types, answers should be listed and separated by a comma only (no spaces) and be preceeded by 'choices='. For example 'choices=Yes,No'.
+  - For numeric answer types, specify the minimum value using 'min=' and the size of the step using 'step=', with these separated by a semicolon and no spaces. For example 'min=1;step=0.5'.
```

Comment:
Does this mean that all (non-bounded) numeric answer will be displayed with a slider too?